tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -404,6 +404,9 @@ Status GetTrtBroadcastShape(const TRT_TensorOrWeights& operand_l,
   // Compare broadcast feasibility
   if (check_feasibility) {
     for (int i = 0; i < broadcast_num_dims; ++i) {
+      if (!use_implicit_batch && (output_l[i] == -1 || output_r[i] == -1)) {
```

Comment:
Can we add comment here to explain the decision here? In particular, why we allow cases like these:
   (output_l[i], output_r[i]) = (-1, -1), (40, -1)
With implicit mode, we don't have (-1, -1), we only support (-1, 1) or (1, -1).